### ISSUE TITLE
[Bug] Text in "find a course" search box is dark when it should be white

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1581]
### Currently

![Screen Shot 2021-04-12 at 3.23.59 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/4fb93c55-ebe1-4f10-9d35-99caa7a82e34)

[Study homepage](https://study.unimelb.edu.au)

Label text is showing up with new specificity/colour after recent global changes to base font colours.

### Proposed fix

- [x] Ensure text appears as white in this case

**Comment by gepifanio** [comment]
PR: https://github.com/unimelb/pattern-lib/pull/1582
